backend/src/game/dice_handler.py

```diff
@@ -0,0 +1,26 @@
+import backend.src.game.dice as dice
+import backend.src.game.constants as constants
+
+
+class DiceHandler:
+    dice_values = []
+    re_rolls_left = constants.DEFAULT_RE_ROLL_COUNT
+
+    def start_turn(self):
+        self.dice_values = dice.roll_many(constants.DEFAULT_DICE_COUNT)
+        self.re_rolls_left = constants.DEFAULT_RE_ROLL_COUNT
+
+    def re_roll_dice(self, indexes_of_dice_to_re_roll):
```

Comment:
This should really only accept one type of input and then you would not need type checking, which exposes you to more possible issuses later on.

Reply:
Thanks!